### ISSUE TITLE
Support "-" for X-Forwarded-Proto log for Gorouter

### ISSUE THREAD

**Issue by mike1808** [dmikusa/access_log_parser#1]
Whenever `X-Forwarded-Proto` header is `"-"` the log cannot be parsed.

Example:

```
   Code(
        "-\" x_forwarded_proto:\"-\" vcap_request_id:\"eb922abf-0f2d-4042-4a84-9161e6ee17a1\" response_time:0.000108962 app_id:\"-\" app_index:\"-\" x_b3_traceid:\"81aa595b268bbe68\" x_b3_spanid:\"81aa595b268bbe68\" x_b3_parentspanid:\"-\" b3:\"81aa595b268bbe68-81aa595b268bbe68\"",
        Tag,
    ),

```
for 

```
35.243.162.217:80 - [2019-10-31T00:11:09.329+0000] "GET / HTTP/1.1" 404 0 69 "-" "Mozilla/5.0 (Windows NT 10.0; WOW64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/52.0.2743.116 Safari/537.36" "185.89.158.86:36539" "-" x_forwarded_for:"-" x_forwarded_proto:"-" vcap_request_id:"eb922abf-0f2d-4042-4a84-9161e6ee17a1" response_time:0.000108962 app_id:"-" app_index:"-" x_b3_traceid:"81aa595b268bbe68" x_b3_spanid:"81aa595b268bbe68" x_b3_parentspanid:"-" b3:"81aa595b268bbe68-81aa595b268bbe68"
```

**Comment by dmikusa** [comment]
OK, thanks for the heads up. I think I can fix this one pretty easy. Will keep you posted.

**Comment by dmikusa** [comment]
When I get a chance to fix #2 I'll push out a new release of `top-logs`.
